app/page.tsx

```diff
@@ -1,3 +1,14 @@
+import Hero from '@/components/Hero';
+import HeroChild from '@/components/HeroChild';
+import FeaturesComponent from '@/components/Features';
+
 export default function Home() {
-  return <div></div>;
+  return (
+    <div>
+      <Hero largeText="Event registration simplified for organisers and attendees.">
```

Comment:
Can add the catchy phrase u said: "Canva for event registrators". :)

Reply:
I'll create a separate pull request to make this change alongside other changes since a lot of the pull requests are dependent on each other!